python/paddle/fluid/tests/custom_op/CMakeLists.txt

```diff
@@ -19,10 +19,21 @@ if(WITH_GPU OR APPLE)
   endif()
 endif()
 
+if(WITH_GPU)
+  py_test(test_custom_op_relu_model_static_multidevice
+          SRCS test_custom_op_relu_model_static_multidevice.py)
+  set_tests_properties(test_custom_relu_model
```

Comment:
`test_custom_relu_model `->`test_custom_op_relu_model_static_multidevice`

Reply:
Done. And Thanks.